### PR TITLE
stack network module for 2018-03-01-hybrid profile

### PR DESCRIPTION
**Pull request by bganapa**
## Description

stack specific network module , copy of AzureRm.Network module 5.0.0 , The already published module uses 2017-09-01 api version and we need 2017-10-01 api version. we just made a copy of 5.0.0 and changed the SDK dll 

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
@bganapa Looks like Credscan is failing, please check out the errors here: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell/6937/artifact/src/Package/CredentialScannerOutput/CredScanIssues.csv/*view*/

File: src/StackAdmin/Network/ChangeLog.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+        - Additional information about change #2
+    * Overview of change #3
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Current Release
```
Comment (bganapa) [inline src/StackAdmin/Network/ChangeLog.md]: Keep only current as this is tack specific module and change descriptions

File: src/StackAdmin/Network/Commands.Network.Test/Commands.Network.Test.csproj [hunk 2]
```diff
@@ -43,43 +43,201 @@
     <RunCodeAnalysis>false</RunCodeAnalysis>
   </PropertyGroup>
   <ItemGroup>
+<!--    <Reference Include="Hyak.Common">
```
Comment (bganapa) [inline src/StackAdmin/Network/Commands.Network.Test/Commands.Network.Test.csproj]: commented out code

File: src/StackAdmin/Network/Commands.Network.Test/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -32,6 +32,6 @@
 
 [assembly: Guid("d1e9969e-7b83-4dfc-a7a2-523d4f7e6cc9")]
 
-[assembly: AssemblyVersion("4.3.2")]
-[assembly: AssemblyFileVersion("4.3.2")]
+[assembly: AssemblyVersion("5.0.0")]
```
Comment (bganapa) [inline src/StackAdmin/Network/Commands.Network.Test/Properties/AssemblyInfo.cs]: 5.0.1

File: src/StackAdmin/Network/Commands.Network.Test/Commands.Network.Test.csproj [hunk 4]
```diff
@@ -43,43 +43,201 @@
...
+    <Reference Include="Microsoft.Azure.Management.Compute">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Compute.17.0.0\lib\net452\Microsoft.Azure.Management.Compute.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
     <Reference Include="Microsoft.Azure.Management.Redis">
-      <HintPath>..\..\..\..\packages\Microsoft.Azure.Management.Redis.3.1.1-preview\lib\net45\Microsoft.Azure.Management.Redis.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Redis.3.1.1-preview\lib\net45\Microsoft.Azure.Management.Redis.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+<!--    <Reference Include="Microsoft.Azure.ResourceManager">
```
Comment (deathly809) [inline src/StackAdmin/Network/Commands.Network.Test/Commands.Network.Test.csproj]: Commented out code.

File: src/StackAdmin/Network/Commands.Network.Test/Commands.Network.Test.csproj [hunk 5]
```diff
@@ -245,31 +560,84 @@
...
+    </None>
+    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.VirtualNetworkTests\TestVirtualNetworkUsage.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\Compute\Commands.Compute\Commands.Compute.csproj">
-      <Project>{52643bd5-6378-49bd-9f6e-dac9dd8a867b}</Project>
-      <Name>Commands.Compute</Name>
+<!--    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication.Abstractions\Commands.Common.Authentication.Abstractions.csproj">
```
Comment (deathly809) [inline src/StackAdmin/Network/Commands.Network.Test/Commands.Network.Test.csproj]: Commented out code.

File: src/StackAdmin/Network/Commands.Network/Commands.Network.csproj [hunk 6]
```diff
@@ -50,19 +50,26 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="AutoMapper">
-      <HintPath>..\..\..\packages\AutoMapper.3.1.1\lib\net40\AutoMapper.dll</HintPath>
+    <Reference Include="AutoMapper, Version=6.0.2.0, Culture=neutral, PublicKeyToken=be96cd2c38ef1005, processorArchitecture=MSIL">
+      <HintPath>$(LibraryNugetPacakageFolder)\AutoMapper.6.0.2\lib\net45\AutoMapper.dll</HintPath>
```
Comment (maddieclayton) [inline src/StackAdmin/Network/Commands.Network/Commands.Network.csproj]: "Package" not "Pacakage" - apply everywhere

**Comment by bganapa** [comment]
looking at the build failure

**Comment by bganapa** [comment]
@maddieclayton updated versions and fixed the conflicts. Thanks again for being very responsive!

**Comment by maddieclayton** [comment]
No problem! Once sign jobs are both passing and @deathly809 signs off we can merge.

**Comment by deathly809** [comment]
@maddieclayton  looks good to me.

**Comment by bganapa** [comment]
psnetsign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/158/
ps-sign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1179/

**Comment by bganapa** [comment]
ps-sign has failed, looking at it

**Comment by bganapa** [comment]
ps-sign is successful now
https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1194/
psnetsign successful as well
https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/168/
@maddieclayton

**Pull request merged**